bazel/thirdparty/libpciaccess.BUILD

```diff
@@ -0,0 +1,75 @@
+#
+# This build is a translation of the libpciaccess meson build:
+# https://gitlab.freedesktop.org/xorg/lib/libpciaccess/-/blob/2ec2576cabefef1eaa5dd9307c97de2e887fc347/meson.build
+#
+cc_library(
+    name = "libpciaccess",
+    srcs = [
+        "src/common_bridge.c",
+        "src/common_capability.c",
+        "src/common_device_name.c",
+        "src/common_init.c",
+        "src/common_interface.c",
+        "src/common_io.c",
+        "src/common_iterator.c",
+        "src/common_map.c",
+        "src/linux_devmem.h",
+        "src/pciaccess_private.h",
+    ] + select({
+        "@platforms//os:linux": [
```

Comment:
Is this worth it considering we only support linux builds?

Reply:
probably not. the reason it is there is because of an aspiration to push some of these back into BCR.